content/en/docs/concepts/signals/traces.md

```diff
@@ -187,6 +187,7 @@ work or operation. Spans are the building blocks of Traces. In OpenTelemetry,
 they include the following information:
 
 - Name
+- Parent span ID
```

Comment:
```suggestion
- Parent span ID (can be null or empty, indicating a root span)
```

Reply:
How about

- Parent span ID (for non-root spans)

Reply:
So even with root spans there is an ID. It's just empty.

Reply:
Ok, how about:

```suggestion
- Parent span ID (empty for root spans)
```